target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -139,7 +139,8 @@ tplink,tl-wr1043n-v5)
 	ucidef_set_led_switch "lan3" "LAN3" "tp-link:green:lan3" "switch0" "0x04"
 	ucidef_set_led_switch "lan4" "LAN4" "tp-link:green:lan4" "switch0" "0x02"
 	;;
-tplink,archer-c6-v2)
+tplink,archer-c6-v2|\
+tplink,archer-a6-v2)
```

Comment:
Apply correct sorting, please (you will need to move the whole block upwards, too).

Reply:
back to c6-v2-us, so it is fine to keep in same position?

Reply:
Yes, indeed :-)